lib/logger/utils.spec.ts

```diff
@@ -0,0 +1,35 @@
+import bunyan from 'bunyan';
+import { validateLogLevel } from './utils';
+
+describe('logger/utils', () => {
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it('checks for valid log levels', () => {
+    expect(validateLogLevel(undefined)).toBeUndefined();
+    expect(validateLogLevel('')).toBeUndefined();
+    expect(validateLogLevel(' ')).toBeUndefined();
+    expect(validateLogLevel('warn')).toBeUndefined();
+    expect(validateLogLevel('debug')).toBeUndefined();
+    expect(validateLogLevel('trace')).toBeUndefined();
+    expect(validateLogLevel('TRACE')).toBeUndefined();
+    expect(validateLogLevel('info' as bunyan.LogLevel)).toBeUndefined();
```

Comment:
```suggestion
  it.each`
    input
    ${undefined}
    ${''}
    ${'warn'}
    ${'debug'}
    ${'trace'}
    ${'TRACE'}
    ${'info'}
  `('checks for valid log level: $input', (input) =>{
    expect(validateLogLevel(input)).toBeUndefined();
```

Reply:
For some reason the undefined is being passed as an object and the test is failing with this change. Lets keep it simple as it is ?